webui/Caddyfile

```diff
@@ -78,6 +78,12 @@ proxy /api/likert_surveys havenapi:3000 {
   transparent
   header_upstream X-Forwarded-Proto {>X-Forwarded-Proto}
 }
+
+proxy /api/likert_questions havenapi:3000 {
```

Comment:
at this point can we collapse all the proxy directives so that everything under /api/ goes to havenapi, or is there still another API in PostgREST?

Reply:
Oh good point. I will test and confirm.

Reply:
@statik we cannot remove the proxy to api just yet since we still need to move ipsative_data over to havenapi as well. I will create a task for that and work on it next.

Reply:
We also need to move the files api still it looks like @statik